lib/AvatarGroup/styles.scss

```diff
@@ -0,0 +1,3 @@
+.avatar-group-list {
+  margin-right: 1em;
```

Comment:
Ideal we'd use a spacing setting e.g. `$layout-spacing-base` but merge away as this will be cleaned up in another PR around Avatars.